### PR TITLE
Add `dev.fe-helpers` to expose the current query to CLJS REPLs

### PR DESCRIPTION
**Pull request by bshepherdson**

### CODE REVIEW

**Comment by bshepherdson** [comment]
Current dependencies on/for this PR:
* master
  * **PR #34997** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34997" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metabase/metabase/34997?utm_source=stack-comment).

File: frontend/src/metabase-lib/Question.ts [hunk 1]
```diff
@@ -1221,6 +1221,12 @@ class Question {
       );
     }
 
+    // Helpers for working with the current query from CLJS REPLs.
+    if (process.env.NODE_ENV === "development") {
+      window.__MLv2_metadata = metadata;
```
Comment (metamben) [inline frontend/src/metabase-lib/Question.ts:1226]: How do you access this in the REPL?

**Review by metamben — APPROVED** [review]
LGTM

File: frontend/src/metabase-lib/Question.ts (hunk 1 above)
Comment (bshepherdson) [reply frontend/src/metabase-lib/Question.ts:1226]: With `(dev.fe-helpers/current-query)`. See https://github.com/metabase/metabase/pull/34997/files#diff-5e5e26a1cd736407870cb6c806d437ee1698e39a4d6f662c32db2f496c3a3c0dR14 which is accessing this hack on `window`.

**Review by kulyk — APPROVED** [review]
LGTM. Be aware that if there're multiple instances of `Question`, these new references on `window` might be overwriting one another

**Pull request merged**